capstone/src/main/webapp/script.js

```diff
@@ -43,4 +43,15 @@ function capitalizeFLetter(string) {
         return "Missing Input";
     }
     return string.replace(/^./, string[0].toUpperCase()); 
+}
+
+function deleteFromDatastore(object) {
+    var keyData = JSON.stringify({"kind": object.entity.key.kind, "id": object.entity.key.id});
+    fetch("/delete", {method: "POST", headers: new Headers({ID_TOKEN}), body: keyData}).then((resp) => {
+        if (resp.ok){
+            location.reload();
+        } else {
+            alert("Error has occured");
```

Comment:
Its important that errors are always as informative as possible. If someone files a bug with your software which say "I got an alert saying 'Error has occurred'" you dont really have anything to go off of. Its also confusing as a user when there is no detail given, because how will I know what to do now?